eng/common/pipelines/templates/steps/git-push-changes.yml

```diff
@@ -8,7 +8,8 @@ parameters:
   WorkingDirectory: $(System.DefaultWorkingDirectory)'
   ScriptDirectory: eng/common/scripts
   SkipCheckingForChanges: false
-
+  PushForNoChanges: false
```

Comment:
We shouldn't need PushForNoChanges parameter now with the other update.

Reply:
Talked offline, and we will move to use the existing paramter.